Description: fix atime comparisons in utime test
 When I look at the source code in test/utime.c, that test
 (and the following one) looks backwards. Computers are fast
 enough that the usual case is st.st_atime == now.
From: Larry Doolittle <larry@doolittle.boa.org>
Forwarded: not-yet

--- a/test/utime.c
+++ b/test/utime.c
@@ -28,8 +28,8 @@ int main(void)
   assert(stat(file, &st) == 0);
 
   assert(st.st_atime == st.st_mtime);
-  assert(st.st_atime >= now);
-  assert(st.st_atime - now < 10);
+  assert(st.st_atime <= now);
+  assert(now - st.st_atime < 10);
 
   unlink(file);
 
